### ISSUE TITLE
General Question about the GRPO loss

### ISSUE THREAD

**Issue by dragonlzm** [www-Ye/TimeZero#6]
Hi! First of all, thank you for the great work! Using RL to train the temporal grounding is really fascinating. I have a general question about the GRPO loss used in this [here.](https://github.com/www-Ye/TimeZero/blob/a7c3523367cbff9f166baae10bc34568cffc26f3/src/open_r1/trainer/grpo_trainer_video.py#L525) 

I notice that since the torch.exp(per_token_logps - per_token_logps.detach())` will always be 1, therefore the model will only be updated using the Advantage but not including the policy loss of the RL. This design is slightly different from the one used in the Deepseek-r1. 

I wonder whether you have tried the version that is more similar to GRPO from the deepseek-r1, which may need multiple iterations since it needs to get the 𝜋_{\cita} / 𝜋_{\cita_{old}}. Or do you have any insight into why this (single iteration) may be enough for the temporal grounding setting?

Thank you so much for your help, and I look forward to your reply!

![Image](https://github.com/user-attachments/assets/02d3f82d-57b9-40cb-85b9-ad981f62fb58)
